### PR TITLE
use of a single channel of structured data + waitgroup

### PR DESCRIPTION
**Pull request by ilmanzo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+go-ping-monitor
```
Comment (bryanpedini) [inline .gitignore:1]: 😅 usando `go run .` non l'ho mai inserito...

File: main.go [hunk 2]
```diff
@@ -28,24 +34,32 @@ func usage(errors ...string) {
...
+	pingres := make(chan pingInfo, len(addresses))
+	var wg sync.WaitGroup
+	for _, addr := range addresses {
+		wg.Add(1)
+		go checkSrv(addr, pingres, &wg)
+	}
+	wg.Wait()
+	close(pingres) // qui posso chiudere il chan
+	for res := range pingres {
+		fmt.Printf("ping %s -> rtt %s ms\n", res.host, res.result)
```
Comment (bryanpedini) [inline main.go:46]: Any reason why "rtt"? I would like to display the average ping.

File: .gitignore (hunk 1 above)
Comment (bryanpedini) [inline .gitignore:1]: Always using `go run .` I never got it... 😅

File: main.go (hunk 2 above)
Comment (bryanpedini) [inline main.go:46]: Any reason why "rtt"? I would like to display the average ping.

**Review by bryanpedini — CHANGES_REQUESTED** [review]
Need feedback.

File: .gitignore (hunk 1 above)
Comment (bryanpedini) [reply .gitignore:1]: Well, actually, **this** does not require any change.

Comment (ilmanzo) [reply .gitignore:1]: If someone runs "go build" it will create a binary with that name, so I added the binary in .gitignore

File: main.go (hunk 2 above)
Comment (ilmanzo) [reply main.go:46]: no particular reason, you can put whatever message you prefer.

File: .gitignore (hunk 1 above)
Comment (bryanpedini) [reply .gitignore:1]: Yup, I know, I just didn't think to add this since I usually don't `go build` but `go run .` 😅

File: main.go (hunk 2 above)
Comment (bryanpedini) [reply main.go:46]: Got it!

**Review by bryanpedini — APPROVED** [review]

**Pull request merged**